src/agent/Makefile.am

```diff
@@ -132,37 +132,69 @@ EXTRA_DIST                = \
     otbr-agent.conf         \
     otbr-agent.default      \
     otbr-agent.service.in   \
+    otbr-agent.init.in      \
     $(NULL)
 
 dbusconfdir = $(DBUS_CONFDIR)
 dbusconf_DATA = otbr-agent.conf
 
+if OTBR_HAS_SYSTEMCTL
 systemddir=$(sysconfdir)/systemd/system
 systemd_DATA                        = \
     otbr-agent.service                \
     $(NULL)
+endif
+
+initddir=$(sysconfdir)/init.d
 
 defaultdir=$(sysconfdir)/default
 
-install-data-local: $(DESTDIR)$(defaultdir)/otbr-agent
+uninstall-local: uninstall-default uninstall-initd
```

Comment:
-> `uninstall-local: uninstall-default`
and
move `uninstall-local: uninstall-initd` inside of `if OTBR_HAS_SYSTEMCTL`